File: isegoria/src/main/java/com/eulersbridge/isegoria/network/GeneralInfoResponse.java
package com.eulersbridge.isegoria.network;

import com.eulersbridge.isegoria.models.Country;
import com.squareup.moshi.Json;

import java.util.List;

public class GeneralInfoResponse {

    @Json(name = "countrys")
    public List<Country> countries;

}

File: isegoria/src/main/java/com/eulersbridge/isegoria/network/api/response/GeneralInfoResponse.kt
package com.eulersbridge.isegoria.network.api.response

import com.eulersbridge.isegoria.network.api.model.Country
import com.squareup.moshi.Json

data class GeneralInfoResponse (
    @field:Json(name = "countrys")
    val countries: List<Country>
)
